test/functional/api/v2/puppetclasses_controller_test.rb

```diff
@@ -137,4 +137,13 @@ class Api::V2::PuppetclassesControllerTest < ActionController::TestCase
     end
     assert_response :success
   end
+
+  test "should not remove puppetclass params" do
+    klass = FactoryGirl.create(:puppetclass, :environments => [FactoryGirl.create(:environment)])
+    FactoryGirl.create(:lookup_key, :as_smart_class_param, :puppetclass => klass)
```

Comment:
Change `lookup_key` to `puppetclass_lookup_key`, it was changed in 6b4b489 on develop.